### ISSUE TITLE
Cannot update Impact Pack from Comfy Manager, "No item found with id 'origin/main'"

### ISSUE THREAD

**Issue by LiJT** [ltdrdata/ComfyUI-Impact-Pack#858]
![PixPin_2024-12-16_23-05-14](https://github.com/user-attachments/assets/1558737a-f19c-400e-9fda-841c61b9f0c6)

Update custom node 'ComfyUI Impact Pack'
Update: ['https://github.com/ltdrdata/ComfyUI-Impact-Pack']
No item found with id 'origin/main'
CUSTOM NODE PULL: Fail

I wonder why the main branch of Impact-pack is upper case "Main", instead of "main" like other custom node? 
I cannot update this custom node from comfy manager becuase of this, maybe that is the case?
How to fix this problem? 
Thank you for your work man

**Comment by gabrielhrr** [comment]
same problem

**Comment by andreszs** [comment]
Did that error prevent the update from actually taking place? Maybe you can just ignore that message, here the update worked without errors or warnings.

> ComfyUI: 2890 9a616b 2024-12-04
> Manager: V2.51.8

**Comment by LiJT** [comment]
> Did that error prevent the update from actually taking place? Maybe you can just ignore that message, here the update worked without errors or warnings.
> 
> > ComfyUI: 2890 9a616b 2024-12-04
> > Manager: V2.51.8

Yes, it prevent me from update from Comfy Manager, I can only do Git pull inside Impact Pack node folder, which is annoying, only this node I cannot update automatically. Can we have normal "main" instead of Capital "Main" for the branch?
